.travis.yml

```diff
@@ -10,13 +10,16 @@ notifications:
   email: false
 
 addons:
-  postgresql: "9.4"
+  postgresql: "9.5"
```

Comment:
This is unrelated to the scope of this PR (the `next` branch already sets it to 9.5)